_layouts/estimate.html

```diff
@@ -4,6 +4,6 @@
 
 <section class="estimate">
     <form action="" class="estimate__form">
-        <input type="submit">Submit</input>
+        <input class="estimate__submit" type="submit">
```

Comment:
Those elements should be WCAG 2.0 compliant since it's becoming google standard as we speak.

Reply:
The form is disposable, it was just to test of the submit button redirect in UI.